Makefile

```diff
@@ -57,3 +60,17 @@ generate-hive-templates: generate-oauth-templates
 	else \
 		${GEN_TEMPLATE}; \
 	fi
+
+.PHONY: generate-acm-policies
+generate-acm-policies: generate-acm-policies
```

Comment:
Is this a duplicate or am I just having a hard time reading this?

Reply:
My first time working with makefile so i wasn't sure. I'll fix that !